nautobot/ipam/factory.py

```diff
@@ -449,6 +296,8 @@ class Params:
         factory.SubFactory(VRFGetOrCreateFactory, tenant=factory.SelfAttribute("..tenant")),
         None,
     )
+    rir = factory.Maybe("has_rir", random_instance(RIR, allow_null=False), None)
```

Comment:
Do we have any easy way to tell whether this is a root-level prefix or not? Would it be more "realistic" (maybe??) if only root-level prefixes get RIRs?

Reply:
I updated `PrefixFactory` to set `has_rir=False` on the child prefixes created by `PrefixFactory.children`